sync/src/com/kirinpatel/sync.java

```diff
@@ -0,0 +1,80 @@
+package com.kirinpatel;
+
+import com.kirinpatel.net.User;
+import com.kirinpatel.util.UIMessage;
+import jdk.nashorn.api.scripting.URLReader;
+import uk.co.caprica.vlcj.discovery.NativeDiscovery;
+
+import javax.swing.*;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public final class sync {
+
+    private final static int VERSION = 1;
+    private final static int BUILD = 6;
+    private final static int REVISION = 0;
+    public static long deSyncWarningTime = 1000;
+    public static long deSyncTime = 5000;
+    public static boolean showUserTimes = false;
+    public static ArrayList<User> connectedUsers = new ArrayList<>();
+
+    public static void main(String[] args) {
+        if (isUpdated()) {
+            try {
+                UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+            } catch(Exception e) {
+                UIMessage.showErrorDialog(e, "Unable to set look and feel of sync.");
+            } finally {
+                if (verifyDependencies()) {
+                    Launcher.setInstance();
+                }
+            }
+        } else {
+            UIMessage.showMessageDialog(
+                    "You have an outdated version of sync, please update sync!",
+                    "Outdated version of sync.");
+        }
+    }
+
+    private static boolean verifyDependencies() {
+        if (!new NativeDiscovery().discover()) {
+            UIMessage.showErrorDialog(new IllegalAccessException("Unable to load VLCJ." +

```

Comment:
I am unsure what would be a better exception, but you would not be able to use sync in a state where VLC and Java do not have the same architecture.